.github/workflows/pika.yml

```
@@ -201,21 +204,21 @@
+        working-directory: ${{ github.workspace }}
         run: |
-          chmod +x ../tests/integration/start_master_and_slave.sh
-          ../tests/integration/start_master_and_slave.sh
-          chmod +x ../tests/integration/start_codis.sh
-          ../tests/integration/start_codis.sh
+          cd tests/integration/
+          chmod +x start_master_and_slave.sh
+          ./start_master_and_slave.sh
+          chmod +x start_codis.sh
+          ./start_codis.sh
+          
```

yamllint

[error] trailing spaces

(trailing-spaces)